component/EmojiIcon.js

```diff
@@ -9,7 +9,8 @@ const styles = StyleSheet.create({
         justifyContent: 'center'
     },
     emoji: {
-        textAlign: 'center'
+        textAlign: 'center',
+        color:'#000'
```

Comment:
This change is more specific to your app, probably better to fork this and modify within your app

Reply:
Ive already forked, but this is not an app specific  problem, currently emoji icons are shown with some kind of opacity in some android devices, and by set emoji colors to black this issue will be fixed. this wont change the color of emojis.